pom.xml

```diff
@@ -255,7 +255,7 @@
         <dependency>
             <groupId>org.elasticsearch.client</groupId>
             <artifactId>elasticsearch-rest-high-level-client</artifactId>
-            <version>7.0.1</version>
+            <version>6.6.2</version>
```

Comment:
is there a newer elasticsearch version, which we can use?

Reply:
Yes and no and yes ;):
The currently used grafana version supports only elastic 6.6, but 6 days ago a newly released Grafana version (6.2) was released which supports elastic 7, I do not know how stable this works. Elastic 7 itself already released versions 7.1 and 7.2.

Reply:
please add comment that the version is due to grafana 6.2